bot/cogs/dev_tools.py

```diff
@@ -93,6 +93,12 @@ async def view_test(self, ctx: RoboContext) -> None:
         view = MaybeView(ctx)
         view.message = await ctx.send("yeo", view=view)
 
+    @commands.command(name="webhook-test")
+    async def webhook_test(self, ctx: RoboContext) -> None:
+        url = "yee"
+        webhook = discord.Webhook.from_url(url, session=self.bot.session)
+        await webhook.send("hi there", thread=discord.Object(1182960737476419584))
```

Comment:
What is this magical number?

Reply:
More than likely a thread ID. This command was created when first experimenting with how webhooks work as this feature is completely undocumented. the dpy tags also do not provide much information about webhooks as well